ingestion-beam/src/main/resources/account-ecosystem-ping.schema.json

```diff
@@ -0,0 +1,33 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Format of Account Ecosystem Pings before decryption",
+    "type": "array",
+    "items": {
+        "type": "object",
+        "properties": {
+            "ecosystem_anon_id": {
+                "description": "The current Account Ecosystem Telemetry anon_id value for this user; it must be a JOSE JWE objects in Compact Serialization form",
+                "type": "string"
+            },
+            "previous_ecosystem_anon_ids": {
+                "description": "Previous Account Ecosystem Telemetry anon_id values associated with this user",
+                "type": "array",
+                "items": {
+                    "type": "string"
+                }
+            }
+        },
+        "required": [
+            "ecosystem_anon_id"
+        ],
+        "not": {
```

Comment:
Validation will fail if fields are present with any of the following names. They're all misspellings or they're names for the decrypted fields rather than the expected input encrypted fields.

Reply:
As I'm implementing the decryptor, it turns out this method doesn't work with multiple fields. Validation will only fail if the payload has _all_ of the listed fields. Instead, I'll be including these field names in the main schema like `"ecosystemAnonId": false` which has no effect if the field is not present, but will always fail if the field _is_ present (since no field matches schema `false`).